Classify miss-inplaced tensors in logs.

laithsakka opened the pull request:
Summary:
use signpost logs, 
a followup is to remove the field possibly_missed_reinplacing_opportunities form dynamo compile table.

Differential Revision: D65180194




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang @aakhundov

zou3519 commented on torch/_dynamo/utils.py:
```diff
@@ -143,6 +147,51 @@
 
 
+# Abstraction on top of counters.
+class ReInplaceTrigger(enum.Enum):
```
can you message me a query on the scuba table sometime?

laithsakka replied:
yeh i will share the query once i and it and collect the new data

zou3519 reviewed (approved):
some of the tests are failing. but otherwise this looks reasonable

laithsakka committed:
Classify miss-inplaced tensors in logs. (#139240)

Summary:

use signpost logs, a followup is to remove the field possibly_missed_reinplacing_opportunities form dynamo compile table.

Test Plan: run existing tests

Differential Revision: D65180194
